### PR TITLE
Initially create transactions for offline donations

### PR DESCRIPTION
**Pull request by wwahammy**
Super initial test work on creating transactions for offsite donations

### CODE REVIEW

**Comment by wwahammy** [comment]
@clarissalimab, I think this is ready for review!

File: spec/lib/insert/insert_donation_spec.rb [hunk 1]
```diff
@@ -109,7 +109,7 @@
     end
   end
 
-describe '#with_sepa', pending: 'we dont use SEPA so why have it?' do
+  describe '#with_sepa', pending: 'we dont use SEPA so why have it?' do
```
Comment (clarissalimab) [inline spec/lib/insert/insert_donation_spec.rb]: Let's remove it?

**Review by clarissalimab — APPROVED** [review]
Looks good!

File: spec/lib/insert/insert_donation_spec.rb (hunk 1 above)
Comment (wwahammy) [reply spec/lib/insert/insert_donation_spec.rb]: Done!

**Pull request merged**